x-pack/platform/plugins/shared/ingest_pipelines/public/application/sections/pipelines_list/not_found_flyout.tsx

```diff
@@ -7,37 +7,99 @@
 
 import React, { FunctionComponent } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { EuiFlyout, EuiFlyoutBody, EuiCallOut } from '@elastic/eui';
+import { EuiFlyout, EuiFlyoutBody, EuiCallOut, EuiCode, EuiButton } from '@elastic/eui';
 import { EuiFlyoutHeader, EuiTitle } from '@elastic/eui';
+import { reactRouterNavigate } from '@kbn/kibana-react-plugin/public';
+import { Error, useKibana } from '../../../shared_imports';
+import { getCreatePath } from '../../services/navigation';
+import { getErrorText } from '../utils';
+
+// All pipleines for integrations end in @custom
+const isIntegrationsPipeline = (name: string) => {
+  return name.toLowerCase().endsWith('@custom');
+};
 
 interface Props {
   onClose: () => void;
-  pipelineName: string | string[] | null | undefined;
+  pipelineName: string;
+  error: Error;
 }
 
-export const PipelineNotFoundFlyout: FunctionComponent<Props> = ({ onClose, pipelineName }) => {
+export const PipelineNotFoundFlyout: FunctionComponent<Props> = ({
+  onClose,
+  pipelineName,
+  error,
+}) => {
+  const { history } = useKibana().services;
+  const renderErrorCallOut = () => {
+    if (error.error === 'Not Found' && isIntegrationsPipeline(pipelineName)) {
```

Comment:
is there anyway we can rely on the error code (404) rather than the error string?